crates/holochain/src/core/workflow/app_validation_workflow.rs

```diff
@@ -214,6 +214,14 @@ async fn app_validation_workflow_inner(
     #[cfg(feature = "unstable-warrants")]
     let mut warrant_op_hashes = vec![];
 
+    #[cfg(all(feature = "unstable-warrants", feature = "test_utils"))]
+    let disable_warrant_issuance = conductor
+        .config
+        .conductor_tuning_params()
+        .disable_warrant_issuance;
+    #[cfg(all(feature = "unstable-warrants", not(feature = "test_utils")))]
+    let disable_warrant_issuance = false;
```

Comment:
Warrants issuance should only be disabled in tests and when the param is set to `true`.